apps/emqx/test/emqx_persistent_session_SUITE.erl

```diff
@@ -69,6 +69,12 @@ groups() ->
             {group, snabbkaffe},
             {group, gc_tests}
         ]},
+        {rocks_tables, [], [
```

Comment:
rocksdb_tables?

Reply:
I removed this part, I didn't go with that idea.